### PR TITLE
Weekly table only some html & UsersData JSON

### PR DESCRIPTION
**Pull request by socrateskld**

### CODE REVIEW

File: client/src/App.js [hunk 1]
```diff
@@ -3,13 +3,20 @@ import { Route, Switch } from "react-router-dom";
 import About from "./pages/About";
 import Home from "./pages/Home";
 
+
```
Comment (gregdyke) [inline client/src/App.js:6]: you could restore this file to its previous state:

git checkout main -- client/src/App.js

File: client/src/index.js [hunk 2]
```diff
@@ -3,8 +3,10 @@ import { BrowserRouter } from "react-router-dom";
 
 import App from "./App";
 
+
```
Comment (gregdyke) [inline client/src/index.js:6]: same, no need to have these changes

File: client/src/Data/UsersData.json [hunk 3]
```diff
@@ -0,0 +1,44 @@
+[
```
Comment (gregdyke) [inline client/src/Data/UsersData.json]: I would actually not put this in a json file, and I would call it bookings.json. But as we are going to be removing it anyway soon (and doing fetch from /api/bookings instead), it doesn't matter

**Review by gregdyke — CHANGES_REQUESTED** [review]
add yarn.lock to the .gitignore file

**Comment by socrateskld** [comment]
All the required changes are already merged , so im close this PR.

**Pull request closed without merging**